source/code-snippets/usage-examples/changeStream.js

```diff
@@ -0,0 +1,31 @@
+// ignored first line
+const { MongoClient } = require("mongodb");
+
+// Replace the following with your MongoDB deployment's connection
+// string.
+const uri =
+  "mongodb+srv://<user>:<password>@<cluster-url>?retryWrites=true&w=majority";
+
+const client = new MongoClient(uri);
+
+async function run() {
+  try {
+    await client.connect();
+
+    const database = client.db("sample_mflix");
+    const collection = database.collection("movies");
+
+    // create a change stream on the movies collection
+    const changeStream = await collection.watch();
+    // use the '.next()' iterator to see the change events
```

Comment:
I think the current comment might make sense to someone more familiar with how it works, but what do you think about making it even more clear to a wider audience? How about: "use the next() function on the changeStream iterator to access the next event"

Reply:
Fixed